qemu-cc/qemu-cc.spec

```diff
@@ -0,0 +1,198 @@
+Name: qemu-cc
+Version: v2.9.0+git.a2e6957b6f
+Release: 0
+Source0: %{name}-%{version}.tar.gz
+Source1: qemu-cc-rpmlintrc
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+
+Summary  : OpenBIOS development utilities
+Group    : Development/Tools
+License  : BSD-2-Clause BSD-3-Clause GPL-2.0 GPL-2.0+ LGPL-2.0+ LGPL-2.1
+
+Requires: qemu-cc-bin
+Requires: qemu-cc-data
+Requires: qemu-cc-doc
+BuildRequires : automake
+BuildRequires : bison
+BuildRequires : flex
+BuildRequires : glib2-devel
+BuildRequires : libattr-devel
+BuildRequires : libcap-devel
+BuildRequires : libcap-ng-devel
+BuildRequires : libtool
+BuildRequires : libtool-ltdl-devel
+BuildRequires : libtool
+BuildRequires : m4
+BuildRequires : numactl-devel
+BuildRequires : python-devel
+BuildRequires : zlib
+BuildRequires : pkgconfig(pixman-1)
+Patch1: configure.patch
+
+%undefine _missing_build_ids_terminate_build
+%define  debug_package %{nil}
+
+%description
+===========
+QEMU is a generic and open source machine & userspace emulator and
+virtualizer.
+
+%package bin
+Summary: bin components for the qemu-cc package.
+Group: Binaries
+Requires: qemu-cc-data
+
+%description bin
+bin components for the qemu-cc package.
+
+
+%package data
+Summary: data components for the qemu-cc package.
+Group: Data
+
+%description data
+data components for the qemu-cc package.
+
+%package doc
+Summary: docs for the qemu-cc package.
+Group: Data
+
+%description doc
+Docs for the qemu-cc package.
+
+%prep
+ls -la /home/abuild/rpmbuild/SOURCES/
+
+%setup -q
+%patch1 -p1
+
+%build
+export LANG=C
+%configure --prefix=/usr --disable-tools --disable-libssh2 --disable-tcmalloc --disable-glusterfs \
+	--disable-seccomp --disable-{bzip2,snappy,lzo} --disable-usb-redir --disable-libusb \
+	--disable-libnfs --disable-tcg-interpreter --disable-debug-tcg --disable-libiscsi   \
+	--disable-rbd --disable-spice --disable-attr --disable-cap-ng --disable-linux-aio   \
+	--disable-brlapi --disable-vnc-{jpeg,png,sasl} --disable-rdma --disable-bluez       \
+	--disable-fdt --disable-curl --disable-curses --disable-sdl --disable-gtk           \
+	--disable-tpm --disable-vte --disable-vnc --disable-xen --disable-opengl            \
+	--disable-slirp --enable-trace-backend=nop --enable-virtfs --enable-attr            \
+        --enable-cap-ng --target-list=x86_64-softmmu
+make V=1  %{?_smp_mflags}
+
+%install
+rm -rf %{buildroot}
+make install DESTDIR=%{buildroot} PREFIX=/usr
+mv %{buildroot}/usr/bin/qemu-system-x86_64 %{buildroot}/usr/bin/qemu-cc-system-x86_64
+
+%files
+%defattr(-,root,root,-)
+
+%files bin
+%defattr(-,root,root,-)
+/usr/bin/qemu-ga
+/usr/bin/qemu-cc-system-x86_64
+/usr/bin/virtfs-proxy-helper
+%dir /usr/libexec
+/usr/libexec/qemu-bridge-helper
+
+%files data
+%defattr(-,root,root,-)
+/usr/share/qemu/acpi-dsdt.aml
```

Comment:
This looks wrong to me - `/usr/share/qemu/` will conflict with a distro-packaged version of qemu so you need to add a `--libdir=`, `--libexecdir=` and `--datadir=` as is used in `qemu-lite.spec`.